src/patternfly/components/Select/select.scss

```diff
@@ -356,6 +363,20 @@
     pointer-events: none;
     background-color: var(--pf-c-select__menu-item--disabled--BackgroundColor);
   }
+
+  &.pf-m-selected {
+    --pf-c-select__menu-item--PaddingRight: var(--pf-c-select__menu-item--m-selected--PaddingRight);
+  }
+}
+
+.pf-c-select__menu-item-icon {
+  position: absolute;
+  top: 50%;
```

Comment:
I wonder if, seeing as you have a var for "right" if we should have one for top also even though its a %

Reply:
We can. We would also need one for `translateY()` since those 2 properties do the vertical alignment.